packages/flutter_nekoton_bridge/android/build.gradle

```diff
@@ -58,7 +58,8 @@ apply plugin: 'com.android.library'
 //     }
 // }
 android {
-    compileSdkVersion 31
+    namespace 'com.broxus.flutter_nekoton_bridge.flutter_nekoton_bridge'
```

Comment:
Why you want to add namespace?

Reply:
Because with gradle 8.x.x and AGP 8.x.x namespace is required

Reply:
Yeah, thank you!